src/routes/login/+page.server.js

```diff
@@ -0,0 +1,40 @@
+import {
+    getFormData,
+    checkIfUsernameExists,
+    checkIfPasswordIsCorrect,
+    getUserId,
+    generateUuid,
+    setSession
+} from '$lib/server/account';
+
+/** @type {import('./$types').Actions} */
+export const actions = {
+    default: async ({ request, locals, cookies }) => {
+        const { username, password } = await getFormData(request);
+
+        const check1 = await checkIfUsernameExists(locals, username);
+        if (!check1) {
+            return {
+                success: false,
+                message: 'Username does not exist'
+            };
+        };
+
+        const check2 = await checkIfPasswordIsCorrect(locals, username, password);
+        if (!check2) {
```

Comment:
On a pas besoin de la variable `check2`
```suggestion
        if (!await checkIfPasswordIsCorrect(locals, username, password)) {
```